pkgs/development/tools/misc/chrpath/default.nix

```diff
@@ -1,14 +1,18 @@
-{ lib, stdenv, fetchurl }:
+{ lib, stdenv, fetchurl, autoreconfHook }:
 
 stdenv.mkDerivation rec {
   pname = "chrpath";
-  version = "0.16";
+  version = "0.17";
 
   src = fetchurl {
```

Comment:
nit: can we use fetchFromGitea, otherwise LGTM.

Reply:
Didn't think of that, but not sure I see the benefit of switching. (It can also be done in a later PR.)